src/modules/order.ts

```diff
@@ -27,13 +27,31 @@ export class OrderModule extends BaseModule {
       referralKickback: params.referralKickback,
     });
 
-    return await wallet.sendTx(
-      {
-        typeUrl: CarbonTx.Types.MsgCreateOrder,
-        value,
-      },
-      opts
-    );
+    if (params.setLeverage) {
+      return await wallet.sendTxs([
+        {
+          typeUrl: CarbonTx.Types.MsgSetLeverage,
+          value: {
+            creator: wallet.bech32Address,
+            market: params.market,
+            leverage: params.setLeverage.shiftedBy(18).toString(10),
```

Comment:
tbh haven't tested, but followed the format set in` leverage.ts`

Reply:
test before merging into SDK?

Reply:
actually im not sure what i meant here lol. i tested locally linked sdk :p

Reply:
ok gotta get the testers to try it

Reply:
no i meant dev test locally with `yarn link`

Reply:
yes that's what i meant, i tested with `yarn link`ed sdk, to testnet, leverage got set.